index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylessheet"
+      href="https://necolas.github.io/normalize.css/8.0.1/normalize.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Alejandro Orozco's Portfolio</title>
+  </head>
+
+  <body>
+    <header class="nav">
+      <div>
+        <h3 class="header_Logo">My Logo</h3>
+      </div>
+      <div>
+        <nav class="hamburger">
+          <i class="material-icons">menu</i>
+        </nav>
+      </div>
```

Comment:
Good job creating the navigation, however, I think that the `div` wrapping the logo and the `nav` is unnecessary since they can stand alone and also the logo should have been an `a` tag in order for it to be used to navigate the website from anywhere in the page.